tests/robot/_resources/keywords/json-manipulation.robot

```diff
@@ -0,0 +1,554 @@
+
+                        # Run Keyword only when value Quantity Value is empty
+                        Run Keyword And Return If    $vQvalue=="${EMPTY}"
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.value   ${vQvalue}
+
+                        # Run Keyword only when value Quantity Value is missing
+                        Run Keyword And Return If    $vQvalue=="missing"
+                        ...    Delete Object From Json  ${payload}  $.valueQuantity.value
+
+                        # Else
+                        Run Keyword  
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.value   ${vQvalue}
+
+update Value Quantity Unit
+    [Arguments]         ${vQunit}
+
+                        # Run Keyword only when value Quantity Unit is empty
+                        Run Keyword And Return If    $vQunit=="${EMPTY}"
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.unit   ${vQunit}
+
+                        # Run Keyword only when value Quantity Unit is missing
+                        Run Keyword And Return If    $vQunit=="missing"
+                        ...    Delete Object From Json  ${payload}  $.valueQuantity.unit
+
+                        # Else
+                        Run Keyword  
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.unit   ${vQunit}
+
+update Value Quantity System
+    [Arguments]         ${vQsystem}
+
+                        # Run Keyword only when value Quantity System is empty
+                        Run Keyword And Return If    $vQsystem=="${EMPTY}"
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.system    ${vQsystem}
+
+                        # Run Keyword only when value Quantity System is missing
+                        Run Keyword And Return If    $vQsystem=="missing"
+                        ...    Delete Object From Json  ${payload}  $.valueQuantity.system
+
+                        # Else
+                        Run Keyword  
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.system    ${vQsystem}
+
+update Value Quantity Code
+    [Arguments]         ${vQcode}
+
+                        # Run Keyword only when value Quantity Code is empty
+                        Run Keyword And Return If    $vQcode=="${EMPTY}"
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.code   ${vQcode}
+
+                        # Run Keyword only when value Quantity Code is missing
+                        Run Keyword And Return If    $vQcode=="missing"
+                        ...    Delete Object From Json  ${payload}  $.valueQuantity.code
+
+                        # Else
+                        Run Keyword  
+                        ...    Update Value To Json    ${payload}    $.valueQuantity.code   ${vQcode}
+
+update Data Absent Reason
```

Comment:
Just a Question:

What happens if the databsentreason parameter is NOT present in the JSON beforehand? Will it be added?